llvm/include/llvm/IR/DataLayout.h

```diff
@@ -82,14 +79,11 @@ struct LayoutAlignElem {
 /// Layout pointer alignment element.
 ///
 /// Stores the alignment data associated with a given pointer and address space.
-///
-/// \note The unusual order of elements in the structure attempts to reduce
-/// padding and make the structure slightly more cache friendly.
```

Comment:
The comment goes back to the times when the fields were bitfields. It is no longer relevant.